### PR TITLE
[hi] Add content\hi\docs\setup\best-practices\enforcing-pod-security-standards.md

### PR DESCRIPTION
**Pull request by ShivamTyagi12345**
In continuation to issue https://github.com/kubernetes/website/pull/29414

Localizes content\en\docs\setup\best-practices\enforcing-pod-security-standards.md to hindi
@mittalyashu @anubha-v-ardhan

### CODE REVIEW

File: content/hi/docs/setup/best-practices/enforcing-pod-security-standards.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+reviewers:
+- tallclair
+- liggitt
```
Comment (anubha-v-ardhan) [inline content/hi/docs/setup/best-practices/enforcing-pod-security-standards.md:4]:
```suggestion
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

**Comment by ShivamTyagi12345** [comment]
hey @anubha-v-ardhan ,Must have missed your attention. Do I need to make any changes? I think its ready to be merged

**Review by jayesh-srivastava — CHANGES_REQUESTED** [review]
Line 38 and 39 : This sentence should be framed as : 
`पॉड सुरक्षा मानक प्रवेश नियंत्रक के `audit` और `warn` मोड, मौजूदा वर्कलोड को तोड़े बिना पॉड्स के बारे में महत्वपूर्ण सुरक्षा अंतर्दृष्टि एकत्र करना, आसान बनाते हैं।`

**Comment by divya-mohan0209** [comment]
@ShivamTyagi12345 : Please could you work on the changes suggested by Jayesh above?

**Comment by divya-mohan0209** [comment]
Hello @ShivamTyagi12345 , please reopen the PR when you're ready to work on the suggestions

**Pull request closed without merging**